Tests/WriterTests.swift

```diff
@@ -49,7 +45,7 @@ class ConsoleWriterTestCase: XCTestCase {
         let writer = ConsoleWriter(method: .print)
```

Comment:
Would it be worth it to inject a mock such that this could be actually tested? I'm assuming it's just being manually verified in the console as it's currently written.

I played around with it and I think I have a way to inject a mock without messing up the normal use (and similar for `OSLogWriter`). I can put up a PR for it if you think it'd be worthwhile.